src/savegame_json.cpp

```diff
@@ -2981,6 +2982,13 @@ void Creature::store( JsonOut &jsout ) const
     }
     jsout.member( "effects", tmp_map );
 
+    jsout.member( "damage_over_time_map" );
+    jsout.start_array();
+    for( const damage_over_time_data &DoT : damage_over_time_map ) {
+        DoT.store( jsout );
+    }
+    jsout.end_array();
+
```

Comment:
```suggestion
jsout.member( "damage_over_time_map", damage_over_time_map );
```
this is another bit of code that can be simplified, but you must have a function named `serialize` in the damave_over_time_data class.